### PR TITLE
Improve reporting issue & support docs

### PR DESCRIPTION
**Pull request by geekgonecrazy**
Related to: https://github.com/RocketChat/Rocket.Chat/pull/7492

### CODE REVIEW

File: 1. Contributing/Reporting Issues/README.md [hunk 1]
```diff
@@ -2,29 +2,52 @@
...
-Here are some suggested things to include, when applicable, which may help us troubleshoot a problem:
+If your issue isn't a bug please see [Getting Support](/2. Getting%20Support)
+
+## Reporting a bug
+
+Did you find a bug? Reporting is easy!
+
+1. [Search the Issues](https://github.com/RocketChat/Rocket.Chat/issues) for your bug.  If you find it, give it a thumbsup and add some additional information to the ticket if differing.  This will help give us a fuller picture.
+2. Make sure you are running the latest version of Rocket.Chat. Its very likely simply updating will resolve your issue.
+3. Try to narrow it down to reproducable steps.  If we can reproduce it, we will be able to fix the issue a lot quicker.
```
Comment (gdelavald) [inline 1. Contributing/Reporting Issues/README.md]: > Reproducible

File: 1. Contributing/Reporting Issues/README.md [hunk 2]
```diff
@@ -2,29 +2,52 @@
...
+
+Here are some things to include, when applicable, which may help us troubleshoot a problem:
 
 * Operating System / Version / Architecture (64 bit?)
 * Browser type & Version (if web app), any browser add-ons which may be involved (e.g. AdBlocker, NoScript, etc.)
-* Desktop Environment (if desktop app) / Version
-* Rocket.Chat edition / Version or [build number](/2. Getting%20Support)
+* Rocket.Chat Version
 * Expected behavior vs. Actual behavior (In other words, the "bug")
 * Can it be reproduced? If yes, how?
```
Comment (gdelavald) [inline 1. Contributing/Reporting Issues/README.md]: I think we can add something like
> Can it be reproduced easily and what are the steps to do it? Or it's something that happens now and then?

File: 1. Contributing/Reporting Issues/README.md (hunk 1 above)
Comment (geekgonecrazy) [reply 1. Contributing/Reporting Issues/README.md]: Good catch :+1:

File: 1. Contributing/Reporting Issues/README.md (hunk 2 above)
Comment (geekgonecrazy) [reply 1. Contributing/Reporting Issues/README.md]: I think we partially capture that by asking if reproducible.  I'll adjust.  

I think important thing here is we want them to realize for us to fix it.. we need to be able to reproduce the error

**Review by gdelavald — APPROVED** [review]

**Pull request merged**